npm whoami results in 401 error

```
➜  react-youtube-player git:(master) ✗ npm whoami
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/gajus/.nvm/versions/node/v5.10.0/bin/node" "/Users/gajus/.nvm/versions/node/v5.10.0/bin/npm" "whoami"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.3
npm ERR! code E401

npm ERR! Registry returned 401 for GET on http://127.0.0.1:5080/-/whoami
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/gajus/Documents/dev/gajus/react-youtube-player/npm-debug.log
➜  react-youtube-player git:(master) ✗ npm who
npm ERR! Darwin 15.3.0
npm ERR! argv "/Users/gajus/.nvm/versions/node/v5.10.0/bin/node" "/Users/gajus/.nvm/versions/node/v5.10.0/bin/npm" "who"
npm ERR! node v5.10.0
npm ERR! npm  v3.8.3
npm ERR! code E401

npm ERR! Registry returned 401 for GET on http://127.0.0.1:5080/-/whoami
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/gajus/Documents/dev/gajus/react-youtube-player/npm-debug.log
```

Expected output: My username.

Context: I am authenticated and I am able to install private packages.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,3 +2,4 @@
 
 - fixes issue with request entity too large, by setting the limit of `express-http-proxy` limit to `Infinity`
 - deprecates the support for `node@0.12`
+- fixes issue with not forwarding non-install commands

--- a/lib/index.js
+++ b/lib/index.js
@@ -75,7 +75,9 @@ module.exports = function (argv) {
   }
   app.get('/_skimdb', redirectToSkimdb);
   app.get('/_skimdb*', redirectToSkimdb);
-
+  app.get('/-/*', proxy(FAT_REMOTE, {
+    limit: Infinity
+  }));
   app.get('/', function (req, res) {
     Promise.all([skimLocal.info(), getCount()]).then(function (resp) {
 
@@ -240,10 +242,6 @@ module.exports = function (argv) {
     }).catch(function (err) {
       logger.info(err);
     });
-  });
-
-  app.get('/*', function (req, res) {
-    res.redirect(FAT_REMOTE + req.originalUrl);
   });
 
   app.put('/*', proxy(FAT_REMOTE, {

--- a/package.json
+++ b/package.json
@@ -13,8 +13,8 @@
     "test-cleanup": "rm -f .npmrc",
     "start": "node ./lib/bin.js"
   },
-  "engines" : {
-    "node" : ">= 4"
+  "engines": {
+    "node": ">= 4"
   },
   "author": {
     "name": "Nolan Lawson",
@@ -52,7 +52,7 @@
     "babel-plugin-syntax-async-functions": "^6.3.13",
     "babel-plugin-transform-regenerator": "^6.3.26",
     "babel-preset-es2015": "^6.3.13",
-    "chai": "^3.4.1",
+    "chai": "^3.5.0",
     "child-process-promise": "^1.1.0",
     "denodeify": "^1.2.1",
     "eslint": "^1.10.3",
@@ -60,7 +60,7 @@
     "istanbul-coveralls": "^1.0.3",
     "memdown": "^1.1.0",
     "mkdirp": "^0.5.1",
-    "mocha": "^2.3.4",
+    "mocha": "^3.4.1",
     "ncp": "^2.0.0",
     "nock": "^5.2.1",
     "node-fetch": "^1.3.3",

--- a/test/test.js
+++ b/test/test.js
@@ -16,6 +16,7 @@ var PouchDB = require('pouchdb');
 var memdown = require('memdown');
 var semver = require('semver');
 var should = require('chai').should();
+var expect = require('chai').expect;
 var pkg = require('../package.json');
 
 var WORK_DIR = 'work_dir';
@@ -153,6 +154,13 @@ describe('main test suite', function () {
     var res = (await exec('npm info pouchdb')).stdout;
     res.should.be.a('string');
     res.should.match(/pouchdb/);
+  });
+
+  it('can do `npm view`', async () => {
+    var res = (await exec('npm view moment')).stdout;
+    console.log(res);
+    res.should.be.a('string');
+    expect(res).to.contain(' \n{ name: \'moment\'');
   });
 
   it.skip('does a package with a postinstall step', async () => {

